tests/unit/parser.js

```diff
@@ -6944,6 +6945,7 @@ exports.testStrictDirectiveASI = function (test) {
     .test("'use strict',function fn() {}\nfn();", options);
 
   TestRun(test, 7)
+    .addError(1, "Missing \"use strict\" statement.")
```

Comment:
This one is my bad; as written, that test was clearly invalid. Thanks for correcting it.